invenio_rdm_records/resources/serializers.py

```diff
@@ -19,11 +19,22 @@
 class UIJSONSerializer(JSONSerializer):
     """UI JSON serializer implementation."""
 
+    def icon_options(self, vocabulary_name):
+        """Provides the icon configuration from the vocabulary."""
+        vocabulary_list = Vocabularies.get_vocabulary(
```

Comment:
Couldn't this be merged with the `_serialize_ui_options_from_vocabulary` function? It seems that is already there the option but we chose to return only the `text` key...Maybe we can define a pair of keys to return from the vocabulary parsing...WDYT?